src/machine/machine-varlink.c

```diff
@@ -207,3 +207,25 @@ int lookup_machine_by_name(sd_varlink *link, Manager *manager, const char *machi
         *ret_machine = machine;
```

Comment:
the second commit title: `machine: introduce io.systemd.Machine.Unregister varlink method` or so.